sdk/python/feast/feature_store.py

```diff
@@ -153,6 +166,57 @@ def get_historical_features(
         )
         return job
 
+    @property
+    def project(self) -> str:
+        return "default"
+
+    def materialize(
+        self, feature_views: List[str], start_date: datetime, end_date: datetime
+    ):
+        full_feature_views = []
```

Comment:
Nitpick on the naming: What does `full` represent here?

Reply:
This is a list of the FeatureView objects instead of just the names, and `feature_views` was already taken; I can change if you have a suggestion

Reply:
`feature_views_to_materialize`?